### ISSUE TITLE
"swssloglevel -l SAI_LOG_LEVEL_ERROR -s -c switch" or any SAI API component no longer works in Master branch

### ISSUE THREAD

**Issue by gechiang** [Azure/sonic-swss#1348]
Noticed that there is no way to set the loglevel for SAI anymore in master branch based image.
Tried master.333 as well as master.339 and both are not working…

admin@str-s6000-acs-8:/var/log/swss$ swssloglevel -p
_SAI_API_ACL                  Unknown log level
_SAI_API_BFD                  Unknown log level
_SAI_API_BMTOR                Unknown log level
_SAI_API_BRIDGE               Unknown log level
_SAI_API_BUFFER               Unknown log level
_SAI_API_COUNTER              Unknown log level
_SAI_API_DEBUG_COUNTER        Unknown log level
_SAI_API_DTEL                 Unknown log level
_SAI_API_FDB                  Unknown log level
_SAI_API_HASH                 Unknown log level
_SAI_API_HOSTIF               Unknown log level
_SAI_API_IPMC                 Unknown log level
_SAI_API_IPMC_GROUP           Unknown log level
_SAI_API_ISOLATION_GROUP      Unknown log level
_SAI_API_L2MC                 Unknown log level
_SAI_API_L2MC_GROUP           Unknown log level
_SAI_API_LAG                  Unknown log level
_SAI_API_MACSEC               Unknown log level
_SAI_API_MCAST_FDB            Unknown log level
_SAI_API_MIRROR               Unknown log level
_SAI_API_MPLS                 Unknown log level
_SAI_API_NAT                  Unknown log level
_SAI_API_NEIGHBOR             Unknown log level
_SAI_API_NEXT_HOP             Unknown log level
_SAI_API_NEXT_HOP_GROUP       Unknown log level
_SAI_API_POLICER              Unknown log level
_SAI_API_PORT                 Unknown log level
_SAI_API_QOS_MAP              Unknown log level
_SAI_API_QUEUE                Unknown log level
_SAI_API_ROUTE                Unknown log level
_SAI_API_ROUTER_INTERFACE     Unknown log level
_SAI_API_RPF_GROUP            Unknown log level
_SAI_API_SAMPLEPACKET         Unknown log level
_SAI_API_SCHEDULER            Unknown log level
_SAI_API_SCHEDULER_GROUP      Unknown log level
_SAI_API_SEGMENTROUTE         Unknown log level
_SAI_API_STP                  Unknown log level
_SAI_API_SWITCH               Unknown log level
_SAI_API_SYSTEM_PORT          Unknown log level
_SAI_API_TAM                  Unknown log level
_SAI_API_TUNNEL               Unknown log level
_SAI_API_UDF                  Unknown log level
_SAI_API_VIRTUAL_ROUTER       Unknown log level
_SAI_API_VLAN                 Unknown log level
_SAI_API_WRED                 Unknown log level
buffermgrd                    NOTICE
fpmsyncd                      NOTICE
intfmgrd                      NOTICE
natmgrd                       NOTICE
natsyncd                      NOTICE
nbrmgrd                       NOTICE
neighsyncd                    NOTICE
orchagent                     NOTICE
portmgrd                      NOTICE
portsyncd                     NOTICE
syncd                         NOTICE
teammgrd                      NOTICE
teamsyncd                     NOTICE
tlm_teamd                     NOTICE
vlanmgrd                      NOTICE
vrfmgrd                       NOTICE
vxlanmgrd                     NOTICE
admin@str-s6000-acs-8:/var/log/swss$
admin@str-s6000-acs-8:/var/log/swss$ swssloglevel -l SAI_LOG_LEVEL_ERROR -s -c SWITCH
Component not present in DB

Usage: swssloglevel [OPTIONS]
SONiC logging severity level setting.

Options:
         -h     print this message
         -l     loglevel value
         -c     component name in DB for which loglevel is applied (provided with -l)
         -a     apply loglevel to all components (provided with -l)
         -s     apply loglevel for SAI api component (equivalent to adding prefix "SAI_API_" to component)
         -p     print components registered in DB for which setting can be applied

Examples:
        swssloglevel -l NOTICE -c orchagent # set orchagent severity level to NOTICE
        swssloglevel -l SAI_LOG_LEVEL_ERROR -s -c SWITCH # set SAI_API_SWITCH severity to ERROR
        swssloglevel -l SAI_LOG_LEVEL_DEBUG -s -a # set all SAI_API_* severity to DEBUG
admin@str-s6000-acs-8:/var/log/swss$

admin@str-s6000-acs-8:~$ show vers

SONiC Software Version: SONiC.master.339-31baf381
Distribution: Debian 10.4
Kernel: 4.19.0-6-2-amd64
Build commit: 31baf38
Build date: Tue Jul 7 07:01:13 UTC 2020
Built by: johnar@jenkins-worker-8

Platform: x86_64-dell_s6000_s1220-r0
HwSKU: Force10-S6000
ASIC: broadcom
Serial Number: 1QBRX42
Uptime: 20:32:35 up 1:09, 1 user, load average: 1.12, 1.09, 1.12

**Comment by stephenxs** [comment]
I think this should be the consequence of this issue [357](https://github.com/Azure/sonic-swss-common/issues/357)
someone inserts the log level entries through ProductionStateTable but nobody tries consuming those entries, leaving the keys of most of the entries prefixed with an underscore and failing the swssloglevel who doesn't try to handle this case.

**Comment by gechiang** [comment]
@marian-pritsak Someone told me that Marian implemented this for SAI.  Can Marian please comment as what is needed to fix this issue?
Thanks!

**Comment by gechiang** [comment]
@shi-su How are we doing on this issue?
I have another issue pending on this to be resolved (https://github.com/Azure/sonic-buildimage/issues/4646).
BRCM requires us to set the SAI_API_PORT log level to INFO in order to get more debug information for them to debug the issue.  Appreciate if this log level setting issue can be addressed with a higher priority if possible.
Thanks!

**Comment by shi-su** [comment]
@gechiang I just submitted a PR for this issue. Azure/sonic-swss-common#365
